styles.css

```diff
@@ -0,0 +1,70 @@
+body {
+  background-image: url('img/header-illsutration-mobile.png');
+  background-repeat: no-repeat;
+}
+
+.text_color{
+  color: green;
+}
+
+.header {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 60px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.paragraph {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+  margin-top: -2em;
+}
+
+i {
+  display: flex;
+  align-items: flex-end;
+  margin-left: 95%;
+}
+
+.images {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-evenly;
+  align-items: center;
+  filter: grayscale(100%);
+  list-style-type: none;
+  margin-right: 5em;
+ 
+}
```

Comment:
- [ ] Please try as much as possible to stick to the Figma design, inclusive on the color. I suggest that the icons color match the on of the Figma design to make your project more professional.
Your design 😎 
- [ ] [OPTIONAL] Kindly try as much as possible to change the size of the icons to match the Figma design.

![image](https://user-images.githubusercontent.com/99394374/234807744-d5bd3882-7916-4bb1-8a7d-44215b72cada.png)

Figma design 😎 
![234687075-6ada9af9-eb02-41b5-9597-3c3e20225188](https://user-images.githubusercontent.com/99394374/234807939-2440b2a9-975f-4c62-baa7-da7d84dd4240.png)